packages/app/src/cli/services/dev.ts

```diff
@@ -151,12 +131,44 @@ async function dev(options: DevOptions) {
   }
 }
 
+async function generateFrontendURL(options: DevOptions): Promise<{frontendUrl: string; frontendPort: number}> {
+  let frontendPort: number
+  let frontendUrl: string
+
+  if (options.tunnel === false) {
+    frontendPort = await port.getRandomPort()
+    frontendUrl = 'http://localhost'
+  } else if (options.tunnelUrl) {
```

Comment:
In recent testing of 1P apps from a local environment I realized there's a need for passing the port without a tunnel URL (i.e. `--port`). This is the case locally because `dev` runs the processes in a virtualized environment and opens a port to access it from the host. That being said, I think we should break the convention here and require users to use two flags: `--tunnel-url https://my-tunnel-url.com --port 12345`